sonar-text-plugin/src/test/java/org/sonar/plugins/common/TextAndSecretsSensorTest.java

```diff
@@ -73,20 +73,6 @@ void describe() {
     assertThat(logTester.logs()).isEmpty();
   }
 
-  @Test
```

Comment:
Why is this removed? Was it removed automatically when running the script to create the tests?

Reply:
No, I removed it because it is an artifact from the beginnings of sonar-text